conf/messages.en

```diff
@@ -427,23 +427,16 @@ partyTypeMismatch.body.p2 = Check that you have chosen the right entity type and
 
 detailsDoNotMatch.title = The details you have entered do not match our records
 detailsDoNotMatch.heading = The details you have entered do not match our records
-detailsDoNotMatch.telephone.uk.label = Telephone:
-detailsDoNotMatch.textphone.label = Textphone:
-detailsDoNotMatch.telephone.intl.label = Outside UK:
-detailsDoNotMatch.open.label = Opening times:
-detailsDoNotMatch.open = Monday to Friday, 8am to 6pm
-detailsDoNotMatch.closed = Closed on weekends and bank holidays.
-
-detailsDoNotMatch.ct.body.p1 = Check your company''s Corporation Tax records and try again.
-detailsDoNotMatch.ct.body.p2 = If you are still having problems, contact the Corporation Tax helpline.
+
+detailsDoNotMatch.ct.body.p1 = Check your Self Assessment records and try again.
```

Comment:
Should the wording not mention Corporation Tax rather than Self Assessment?

Reply:
Confluence states that it's a Self Assessment. After task is done we can reiterate this with Content team ?

Reply:
Yeah, I'm not sure I agree with Confluence. This page is for entities using corporation tax (hence 'ct' mentioned everywhere in the code).

Reply:
This has been addressed now.